src/main/java/net/minestom/server/network/packet/server/play/NamedSoundEffectPacket.java

```diff
@@ -9,15 +9,20 @@
 public class NamedSoundEffectPacket implements ServerPacket {
 
     public String soundName;
-    public SoundCategory soundCategory;
+    public int soundCategory;
```

Comment:
Either use Source or SoundCategory, but don't replace an enum by an int inside packets!
We can use Java's compiler to ensure the given category is valid thanks to types.

Reply:
So the main reason that I removed a lot of the enums from these packets was to make the packets more raw, thus allowing for further customisation and control by developers. For example, modded sound categories would require changing the enum but with this method they could be done on the fly if needs be. I suppose the enum could always be edited, but there are many places where raw values are taken in the packet.

Reply:
As per discussion in Discord, this has been reverted to use enum.